test/HealthChecks.Azure.Data.Tables.Tests/AzureTableStorageHealthCheckTests.cs

```diff
@@ -129,47 +119,44 @@ await enumerator
     }
 
     [Fact]
-    public async Task return_unhealthy_when_checking_unhealthy_container()
+    public async Task return_unhealthy_when_checking_unhealthy_service_queue()
```

Comment:
queue? did you mean "table"?

```suggestion
    public async Task return_unhealthy_when_checking_unhealthy_table()
```